jinahub/indexers/dbms/LMDBDBMSIndexer/__init__.py

```diff
@@ -0,0 +1,97 @@
+import os
+from typing import Dict
+
+from jina import Executor, Document, DocumentArray, requests
+
+import lmdb
+
+
+class LMDBDBMSIndexer(Executor):
```

Comment:
I would also add the `dump` method, even thought we might not need to call it on `close`. Could this be the reason, why the `FileBasedIndexer` is soo much slower, since it dumps at the end?

Reply:
FileDBMS doesn't dump by default, so the comparison holds